lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -145,4 +145,26 @@ def table_alias_length
     63
   end
   alias index_name_length table_alias_length
+
+
+  private
+
+  # Extracts the value from a PostgreSQL column default definition.
+  # Some of postgresql's default regexps are not going to work with cockroachdb,
+  # leaving the default values out of the schema.rb file.
+  def extract_value_from_default(oid, default) # :nodoc:
+    ret = super
+    if ret.nil?
+      ret = case default
+        # Numeric types are identified in a different way by cockroachdb
```

Comment:
Could you also put an example of a value that would fail before this into the comment? As a reader, it's not clear immediately what this regex is capturing.

(Only after comparing this to the postgres numeric regex, I know the difference here is INT vs. bignum)